<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>order</class>
 <widget class="QWidget" name="order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTabWidget" name="food1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>911</width>
     <height>551</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 205, 123);
background-color: rgb(224, 112, 0);
background-color: rgb(137, 206, 0);
background-color: rgb(85, 255, 255);
border-right-color: rgb(255, 198, 84);
selection-color: rgb(170, 170, 127);
border-color: rgb(255, 204, 74);
background-color: rgb(170, 255, 127);
background-color: rgb(239, 120, 0);
background-color: rgb(226, 113, 0);
background-color: rgb(175, 88, 0);</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Main</string>
    </attribute>
    <widget class="QTextBrowser" name="food11">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="food11c">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>120</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QTextBrowser" name="food12">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="food12c">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>270</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QTextBrowser" name="food13">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="food13c">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>120</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QTextBrowser" name="food14">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>160</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="food14c">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>270</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 157, 19);
background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="pic.qrc">:/chaofan.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="pic.qrc">:/chaofen.jpeg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="pic.qrc">:/jiaozi.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>160</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="pic.qrc">:/tofu.jpeg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>420</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Add To Cart</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Appetizers</string>
    </attribute>
    <widget class="QTextBrowser" name="food16">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>171</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="food16c">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>130</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="pic.qrc">:/vegieSoup.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>420</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Add To Cart</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Drinks</string>
    </attribute>
    <widget class="QTextBrowser" name="food15">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial Rounded MT Bold'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="pic.qrc">:/drinks.jpeg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="food15_c">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>420</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Add To Cart</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>-10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>28</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 11pt &quot;Segoe UI&quot;;
font: 48pt &quot;Segoe UI&quot;;
font: 28pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
